Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: build up down
+
+build:
+	docker-compose build
+
+up:
```

Comment:
How is IMAGE_TAG supplied?  Run deploy.sh instead??

Reply:
Added both options